lms/djangoapps/teams/tests/test_views.py

```diff
@@ -1299,7 +1300,8 @@ def setUp(self):  # pylint: disable=arguments-differ
         ('student_enrolled', 403),
         ('staff', 204),
         ('course_staff', 204),
-        ('community_ta', 204)
+        ('community_ta', 204),
+        ('admin', 204)
```

Comment:
What does 204 status means? is it HTTP 204 no content?

Reply:
Yes, the success URL that doesn't redirect the client anywhere.